WMF Framework/Fetcher.swift

```diff
@@ -68,8 +68,8 @@ open class Fetcher: NSObject {
         }
     }
     
-    @objc(performTokenizedMediaWikiAPIPOSTWithTokenType:toURL:withBodyParameters:cancellationKey:completionHandler:)
-    @discardableResult public func performTokenizedMediaWikiAPIPOST(tokenType: TokenType = .csrf, to URL: URL?, with bodyParameters: [String: String]?, cancellationKey: CancellationKey? = nil, completionHandler: @escaping ([String: Any]?, HTTPURLResponse?, Error?) -> Swift.Void) -> CancellationKey? {
+    @objc(performTokenizedMediaWikiAPIPOSTWithTokenType:toURL:withBodyParameters:cancellationKey:reattemptLoginOn401Response:completionHandler:)
+    @discardableResult public func performTokenizedMediaWikiAPIPOST(tokenType: TokenType = .csrf, to URL: URL?, with bodyParameters: [String: String]?, cancellationKey: CancellationKey? = nil, reattemptLoginOn401Response: Bool = true, completionHandler: @escaping ([String: Any]?, HTTPURLResponse?, Error?) -> Swift.Void) -> CancellationKey? {
```

Comment:
This is called by `logout` with `reattemptLoginOn401Response` set to `true` (https://github.com/wikimedia/wikipedia-ios/blob/8e114376e0f926a2299745d7b648ee459596c0e0/Wikipedia/Code/WMFAuthenticationManager.swift#L209) - is it intentional? When I log out via the app UI, I wouldn't expect a login attempt

Reply:
oh good catch will check!